code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -507,7 +507,7 @@
 
 /obj/item/weapon/gun/launcher/m92/afterattack(atom/target, mob/user, flag)
 	if(user.mind?.cm_skills && user.mind.cm_skills.spec_weapons < 0)
-		if(!do_after(user, 8, TRUE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
same as above.